### PR TITLE
Feat/support currency conversion

### PR DESCRIPTION
**Pull request by pitb2022**
https://github.com/thematters/matters-web/issues/2856
https://github.com/thematters/matters-web/issues/2857

### CODE REVIEW

File: src/components/Forms/PaymentForm/PayTo/CurrencyChoice/LikeCoinChoice.tsx [hunk 1]
```diff
@@ -100,7 +105,12 @@ const LikeCoinChoice: React.FC<LikeCoinChoiceProps> = ({
         LikeCoin
       </TextIcon>
 
-      <CurrencyFormatter value={formatAmount(balance, 0)} currency="LIKE" />
+      <CurrencyFormatter
+        value={formatAmount(balance, 0)}
+        currency="LIKE"
+        subValue={formatAmount(balance * exchangeRate, 2)}
```
Comment (robertu7) [inline src/components/Forms/PaymentForm/PayTo/CurrencyChoice/LikeCoinChoice.tsx:111]: the spacing seems not right

image

Comment (robertu7) [reply src/components/Forms/PaymentForm/PayTo/CurrencyChoice/LikeCoinChoice.tsx:111]: `value` color shouldn't change on focus/hover state?

image

File: src/views/Me/Wallet/index.tsx [hunk 2]
```diff
@@ -65,9 +96,17 @@ const Wallet = () => {
             balanceHKD={balanceHKD}
             canPayout={canPayout}
             hasStripeAccount={hasStripeAccount}
+            currency={currency}
+            exchangeRate={exchangeRateHKD?.rate || 0}
+          />
+          <LikeCoinBalance
```
Comment (robertu7) [inline src/views/Me/Wallet/index.tsx:102]: Same as above, the spacing issue

image

**Review by robertu7 — CHANGES_REQUESTED** [review]

File: src/components/Forms/PaymentForm/PayTo/CurrencyChoice/LikeCoinChoice.tsx (hunk 1 above)
Comment (pitb2022) [reply src/components/Forms/PaymentForm/PayTo/CurrencyChoice/LikeCoinChoice.tsx:111]: The design Draft changed color
ref:https://www.figma.com/file/oGew0i2s0N8IBm9tCpoLLI/%E8%BF%AD%E4%BB%A3%E8%A8%AD%E8%A8%88%E4%B8%AD%E7%B9%BC%E7%AB%99?node-id=7481%3A51539&t=mi8BODNB3zHA5yAR-4

File: src/components/CurrencyFormatter/styles.css [hunk 3]
```diff
@@ -10,6 +10,7 @@
   }
 
   & .subCurrency {
```
Comment (robertu7) [inline src/components/CurrencyFormatter/styles.css:12]: add transition `@mixin all-transition;` to `.subCurrency` and `.currency`?

**Review by robertu7 — APPROVED** [review]

**Pull request merged**